website/source/docs/configuration/client.html.md

```diff
@@ -158,6 +162,8 @@ the full list.
 
 ### `options` Parameters
 
+~> Note: client configuration options for drivers will soon be deprecated.. Please use [plugin options][plugin-options] instead. See the [plugin stanza][plugin-stanza] documentation for more information.
```

Comment:
It seems a bit strange to send them to `plugin options` which just sends them to `plugin stanza` as well? Maybe just remove the middle sentence or maybe I'm missing something.

```suggestion
~> Note: client configuration options for drivers will soon be deprecated. See the [plugin stanza][plugin-stanza] documentation for more information.
```